templates/config.json

```diff
@@ -609,6 +659,10 @@
       "provider": "KnowBe4",
       "summary": "Employees and contractors receive ongoing security awareness training at least annually.",
       "guidance": "Ongoing security awareness training is foundational to the overall risk reduction for any organization, not to mention is a compliance requirement almost universally. Our recommendation is to leverage a vendor, Ataata, who provides a low-cost and engaging way to keep your folks trained. All it takes is less than 5 minutes of everyone's time to watch a fun video and take a quiz every month.",
+      "instructionsSummary": "Confirm/tailor security awareness platform.",
```

Comment:
Another example here. This can be removed since it's repetitive to what's already covered in `instructions`